### DISCUSSION TITLE
https://plugin-components.vuejs.press/zh/guide/utilities/share.html

### DISCUSSION THREAD

**Discussion by giscus** [vuepress-theme-hope/giscus-discussions#828] [Announcements]
# https://plugin-components.vuejs.press/zh/guide/utilities/share.html

分享组件，将页面内容分享到社交媒体。

https://plugin-components.vuejs.press/zh/guide/utilities/share.html

**Comment by actionee** [comment]
可以增加分享到B站动态的组件吗？

**Reply by Mister-Hope** [reply]
有相关的文档吗？

**Reply by actionee** [reply]
在 B 站开平文档中心没有找到：https://openhome.bilibili.com/doc
但在非官方的文档里找到了这些：https://socialsisteryi.github.io/bilibili-API-collect/docs/dynamic/publish.html

**Reply by Mister-Hope** [reply]
你最好仔细读读，这是客户端或者网页的 API，意味着用户需要先登录。

**Reply by actionee** [reply]
我不会编程，像vuepress这样文档非常详细的，我还能照猫画虎一下，😂 这么复杂的，我就不弄啦。谢谢大佬关注，打扰啦！

**Comment by ChidaoDouyu** [comment]
skype已停止运营，建议删除支持

**Reply by Mister-Hope** [reply]
最好开个issue或者直接搞个pr不然我怕我忘了
